primitives/src/sequence.rs

```diff
@@ -38,18 +38,29 @@ pub struct Sequence(pub u32);
 impl Sequence {
     /// The maximum allowable sequence number.
     ///
-    /// This sequence number disables absolute lock time and replace-by-fee.
+    /// The sequence number that disables replace-by-fee, absolute lock time and relative lock time.
     pub const MAX: Self = Sequence(0xFFFFFFFF);
     /// Zero value sequence.
     ///
     /// This sequence number enables replace-by-fee and absolute lock time.
     pub const ZERO: Self = Sequence(0);
+    /// The sequence number that disables replace-by-fee, absolute lock time and relative lock time.
+    pub const FINAL: Self = Sequence::MAX;
     /// The sequence number that enables absolute lock time but disables replace-by-fee
     /// and relative lock time.
     pub const ENABLE_LOCKTIME_NO_RBF: Self = Sequence::MIN_NO_RBF;
     /// The sequence number that enables replace-by-fee and absolute lock time but
     /// disables relative lock time.
+    #[deprecated(
+        since = "TBD",
+        note = "This constant has ambiguous name. Please use ENABLE_LOCKTIME_AND_RBF instead."
+    )]
     pub const ENABLE_RBF_NO_LOCKTIME: Self = Sequence(0xFFFFFFFD);
+    /// The maximum sequence number that enables replace-by-fee and absolute lock time but
+    /// disables relative lock time.
+    /// 
```

Comment:
There is trailing whitespace on this line. Apart the from that the PR LGTM.

Reply:
Thanks for the catch! I've updated.